lib/process-services/src/lib/task-list/services/tasklist.service.ts

```diff
@@ -104,7 +104,7 @@ export class TaskListService {
      * Gets tasks matching a query and state value.
      *
      * @param requestNode Query to search for tasks
-     * @param state Task state. Can be "open" or "completed".
+     * @param state Task state. Can be "open" or "completed". State "all" applies to both "open" and "completed" tasks.
```

Comment:
The documentation file defines the state parameter as 
> Task state. Can be **"active"** or "completed". State "all" applies to both "active" and "completed" tasks.

Whereas in code, it is defined as 
> Task state. Can be **"open"** or "completed". State "all" applies to both "open" and "completed" tasks.